lib/net.js

```diff
@@ -323,8 +323,9 @@ util.inherits(Socket, stream.Duplex);
 // Refresh existing timeouts.
 Socket.prototype._unrefTimer = function _unrefTimer() {
   for (var s = this; s !== null; s = s._parent) {
-    if (s[kTimeout])
-      s[kTimeout].refresh();
+    if (s[kTimeout]) {
+      this[kTimeout].refresh();
```

Comment:
This still needs to be changed. Remove the brackets and line 327 should be `s[kTimeout].refresh();`.

Reply:
Oops, sorry I totally missed the `this`. Changing this ASAP.